docs-mslearn/fabric/create-finops-workspace.md

```diff
@@ -0,0 +1,235 @@
+3. Select **Lakehouse**  
+    :::image type="content" source="./media/create-fabric-workspace-finops/fabric-new-lakehouse.png" border="true" alt-text="Screenshot showing the Lakehouse option." lightbox="./media/create-fabric-workspace-finops/fabric-new-lakehouse.png" :::
+
+For more information, see [Create a lakehouse](/fabric/data-engineering/create-lakehouse).
+
+>[!NOTE]
+>Microsoft Fabric [lakehouses](/fabric/data-engineering/lakehouse-overview) are data architectures that allow organizations to store and manage structured and unstructured data in a single location. They use  various tools and frameworks to process and analyze that data. These tools and frameworks can include SQL-based queries and analytics, machine learning, and other advanced analytics techniques.
+
+## Create a shortcut to storage
+
+Shortcuts in lakehouse allow you to reference data without copying it. It unifies data from different lakehouses, workspaces, or external storage, such as Data Lake Gen2 or Amazon Web Services (AWS) S3. You can quickly make large amounts of data available in your lakehouse locally without the latency of copying data from the source.
+
+To create a shortcut, see [Create an Azure Data Lake Storage Gen2 shortcut](/fabric/onelake/create-adls-shortcut).
+
+1. Select the lakehouse of your choice.
+2. Select the ellipsis (**...**) next to **Files**.  
+    You add CSV and Parquet files under **Files**. Delta tables get added under **Tables**.
+3. Select **New shortcut**.  
+    :::image type="content" source="./media/create-fabric-workspace-finops/fabric-new-shortcut.png" border="true" alt-text="Screenshot showing creating a new shortcut in a lakehouse under the Files folder." lightbox="./media/create-fabric-workspace-finops/fabric-new-shortcut.png" :::
+4. Select **Azure Data Lake Storage Gen 2** and provide the following settings.
+    1. URL = **Data Lake Storage** URL of the Data Lake storage account. See the following note about authentication.
+    2. Connection = **Create a new connection**
+    3. Connection name = &lt;*Any name of your choice*&gt;
+    4. Authentication kind = **Organizational account**
+    5. Sign in when prompted.
+
+Here’s an example screenshot showing the New shortcut connection settings.
+
+:::image type="content" source="./media/create-fabric-workspace-finops/fabric-new-shortcut-connection-settings.png" border="true" alt-text="Screenshot showing the New shortcut connection settings." lightbox="./media/create-fabric-workspace-finops/fabric-new-shortcut-connection-settings.png" :::
+
+>[!IMPORTANT]
+>You must have the *Storage Blob Data Contributor* role when you use *Organizational account* authentication. If you don't have the required permission, you can use other authentication types like *Account key*. Otherwise, you get an `Invalid credentials` error.
```

Comment:
Should this be listed in prereqs? Or maybe as a step for creating the storage account?

Reply:
Moved to prereqs